packages/react-ui/src/presentation/interfaces.ts

```diff
@@ -62,6 +62,10 @@ export interface XCollectionProps<
   children?: React.ReactNode
 }
 
+export interface XEverythingProps extends XCollectionProps {
+  setSelectedSchema: (schemaName: any) => void
```

Comment:
since we're doing this `Everything: (props: HatchifyEverythingProps<TSchemas>) => JSX.Element` there should, be a way for us to use `TSchemas` to type the `schemaName` here, if `TSchemas` is an object of all the partial schemas then i _think_ it can just be `schemaName: keyof schemaName`

Reply:
thanks for the catch! did not mean to leave that